### ISSUE TITLE
racer does not work from commandline

### ISSUE THREAD

**Issue by koyedele** [phildawes/racer#510]
Noob here!

Environment:
MacOSX El Capitan
racer 1.2.5 (installed with cargo)
rust from 7fc4df6d230f81537886f11c57483e0cb61f0948
$RUST_SRC_PATH = /Users/me/sources/rust/src

Expected:
racer complete std::io::B (should show some completions)

Actual:
ERROR:racer::core: read_file couldn't open "/Users/me/sources/rust/src/libstd/../libcore/src/lib.rs". Returning empty string

Looking into the 'libcore' directory, I see there is no 'src' directory in it. Am I doing something wrong?
I worked around this issue by symlinking 'src' back to 'libcore'. Is this the right thing to do?

**Comment by kbknapp** [comment]
You need the Rust sources on your local machine (in `/Users/me/sources/rust/src` as per your setup specified by the `RUST_SRC_PATH` variable). The easiest way to do this,

```
$ cd /Users/me/sources
$ git clone https://github.com/rust-lang/rust
```

Then re-run your example and it should work.

**Comment by phildawes** [comment]
Hi @koyedele , hi @kbknapp. Actually I think this was a problem with racer's cargo.toml handling - it looks like the rust source recently added Cargo.toml files. see #511 
Hopefully this is fixed in racer 1.2.6

**Comment by koyedele** [comment]
Thanks @kbknapp and @phildawes!
